docker-compose.yml

```diff
@@ -51,7 +15,7 @@ rabbitmq:
   volumes_from:
     - data
 mongo:
-  image: "mongo:latest"
+  image: "mongo:4.0.0"
```

Comment:
Why pin to 4?

Reply:
That was in the README. No reason.

Reply:
We should probably use `mongo:4`, which will include all 4.x.x versions.